src/main/java/org/spongepowered/api/entity/EntityTag.java

```diff
@@ -27,6 +27,9 @@
 import org.spongepowered.api.Tag;
 import org.spongepowered.api.util.annotation.CatalogedBy;
 
+/**
+ * Represents a {@link Tag} which targets {@link EntityType}s.
+ */
 @CatalogedBy(EntityTags.class)
 public interface EntityTag extends Tag<EntityType<?>> {
```

Comment:
should be `EntityTypeTag`

Reply:
I didn't call the other ones `BlockTypeTag`, `ItemTypeTag`, `FluidTypeTag`. So why should this one be different?

Reply:
`Entity` != `EntityType`. They're tags of entity types, not entities. "block type" and "block" are interchangeable